src/main/resources/freemarker/owner-dar-approved.html

```diff
@@ -95,13 +95,13 @@
             <td style="border-collapse: collapse; text-align: left; line-height: 25px; display: block; padding: 0px 30px 20px">
                 <table width="600" style="margin: 15px 0 0 0; border-collapse: collapse;">
                     <tr>
-                        <th align="left" style="font-family: 'Roboto', sans-serif; font-style: italic; font-size: 15px; color: #00609f; border-bottom: 1px solid #cccccc; padding: 0 0 5px 5px; text-align: left;">SC ID</th>
+                        <th align="left" style="font-family: 'Roboto', sans-serif; font-style: italic; font-size: 15px; color: #00609f; border-bottom: 1px solid #cccccc; padding: 0 0 5px 5px; text-align: left;">Dataset ID</th>
                         <th align="left" style="font-family: 'Roboto', sans-serif; font-style: italic; font-size: 15px; color: #00609f; border-bottom: 1px solid #cccccc; padding: 0 0 5px 5px; text-align: left;">Dataset Name</th>
                     </tr>
                     <#list dsl>
                         <#items as ds>
                             <tr style="font-family: 'Roboto', sans-serif;">
-                                <td style="font-family: 'Roboto', sans-serif; padding: 5px; border-bottom: 1px solid #cccccc;">${ds.objectId}</td>
+                                <td style="font-family: 'Roboto', sans-serif; padding: 5px; border-bottom: 1px solid #cccccc;">${ds.getDatasetIdentifier()}</td>
```

Comment:
Changes to this class are outside the scope of the PR and should be in your other PR. Wait until this one is merged and then you can simplify your other PR.